zerver/openapi/zulip.yaml

```
@@ -9243,6 +9243,82 @@ paths:
                             "msg": "You don't have permission to delete this message",
                             "result": "error",
                           }
+  /messages/{message_id}/report:
+    post:
+      operationId: report-message
+      summary: Report a message
+      tags: ["messages"]
+      description: |
+        Reports a message to the moderation request channel if the
+        `moderation_request_channel` is configured. Moderators can
+        review the reports and act on the reported content.
+
+        Clients should check for whether the `moderation_request_channel`
+        setting is enabled to decide whether to show the message
+        report option in the UI.
+
+        Clients should also check for whether the user is a non-guest
+        user to show the message report option in the UI.
+
+        If the `report_type` for the report is `"other"`, the `description`
```

IIUC the value is just `other`, with no quotes:
```suggestion
        If the `report_type` for the report is `other`, the `description`
```

There are a few places in the API where we expect a parameter of string type to come JSON-encoded, but it's uncommon.

---

Ok! updated this, i've also updated the doc page screenshot